content/manuals/desktop/release-notes.md

```diff
@@ -23,6 +23,58 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/orgs/docker/projects/51/views/1?filterQuery=) to see what's coming next.
 
+## 4.34.0
+
+{{< release-date date="2024-09-26" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.35.0" build_path="/TBD/" >}}
+
+### New
+
+- WIP
+
+
+### Upgrades
+
+- [Containerd v1.7.21](https://github.com/containerd/containerd/releases/tag/v1.7.21)
+- [Docker Engine v27.3.0](https://docs.docker.com/engine/release-notes/27.3/#2730)
+- [Docker Scout CLI v1.14.0](https://github.com/docker/scout-cli/releases/tag/v1.14.0)
+- Docker Debug `v0.0.37`
+- Linux kernel `v6.10.9`
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed a bug where a proxy in daemon.json would override the proxy set in Docker Desktop.
+- Fixed bug where some Docker subnet ranges were not able to be used.
+- Added Docker VMM, a new powerful backend for creating and managing Docker Desktop Linux VM.
+- Available only on Apple Silicon Macs with macOS 12.5+. Docker VMM unleashes blazing performance like never before.
+- Removed `docker-index` as it is now deprecated.
```

Comment:
this is referring to an executable binary right? Maybe we should briefly clarify what it was / how it's replaced

Reply:
I've updated the description, let me know your thoughts!

Reply:
LGTM